README.md

```diff
@@ -27,41 +43,60 @@ ADAL Android was deprecated on June 2023. We do not support ADAL. See the [ADAL
 
 ### Step 1: Declare dependency on MSAL
 
-Add to your app's build.gradle:
+Add the following dependencies to your app's build.gradle:
 
 ```gradle
 dependencies {
-    implementation 'com.microsoft.identity.client:msal:4.9.+'
```

Comment:
Please add a note for customer to utilize latest version by going to the release page. We sometimes dont update readme with every single major release.

Reply:
Great call-out here, thanks!